localstack/services/cloudwatch/cloudwatch_listener.py

```diff
@@ -0,0 +1,34 @@
+import re
+from datetime import datetime
+from localstack.utils.common import to_str
+from localstack.services.generic_proxy import ProxyListener
+
+
+class ProxyListenerCloudWatch(ProxyListener):
+
+    def return_response(self, method, path, data, headers, response):
+        # Fix Incorrect date format to iso 8601
+        str_content = to_str(response.content)
+        date_tag_regex = '<AlarmConfigurationUpdatedTimestamp>(.*?)</AlarmConfigurationUpdatedTimestamp>'
```

Comment:
nitpick: perhaps we could loop over the tag names here, instead of duplicating (also, there may be more tags being added here in the future:
```
timestamp_tags = ['AlarmConfigurationUpdatedTimestamp', 'StateUpdatedTimestamp']
for tag in timestamp_tags:
    ...
```